libs/common/src/types/mongo-db-models/facility.ts

```diff
@@ -52,6 +52,7 @@ export type Facility = {
   paymentType: string;
   createdAt: UnixTimestamp;
   updatedAt: UnixTimestamp;
+  bondType?: string; // only featured in BOND facilities
   ukefExposure: number;
   guaranteeFee: number;
   submittedAsIssuedDate: UnixTimestampString | null;
```

Comment:
I know this isn't part of your PR, but I noticed in this file we have `dataMigration.drupalFacilityId`

I think this can be removed as it is very very old and there are no instances of `drupalFacilityId`, can we remove please whilst we're here?

Reply:
Ah yes, I'm nervous to remove it as I feel like it might have been added to this file for a reason - it's commented with:
`  // Legacy data from migrating old GEF Facilities into DTFS` - is there any chance instances of this can therefore still exist in the database (despite being super old)?

Reply:
I've asked Abhi/Zain to check the production database to see if any facilities have `drupalFacilityId`

Reply:
There are some old deals with this @BethThomas141 so we should keep it to be safe.

Sorry to muck around!

Reply:
No problem at all, interesting to check! Happy to mark as resolved?